internal/cmd/schema-registry/command_schema_list.go

```diff
@@ -43,6 +43,7 @@ func (c *command) newSchemaListCommand() *cobra.Command {
 	}
 
 	cmd.Flags().String("subject-prefix", "", "List schemas for subjects with a given prefix.")
+	cmd.Flags().Bool("deleted", false, "Include soft-deleted schemas.")
```

Comment:
This is kind of misleading since this shows both non-deleted and soft-deleted schemas.

I chose this to match the flag from `confluent sr subject list --deleted`, which also shows non-deleted and soft-deleted subjects

Reply:
Several other commands have an `--all` flag. Should we do the same here or would that be too vague?

Reply:
Perfect!